### PR TITLE
Added fsx test for exercising file system

### PR DESCRIPTION
**Pull request by harish-24**
This test uses system calls like write, read, mmap
etc. to exercise file system

Signed-off-by: Harish <harish@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
@harish-24  there was some issue in travis , now issue   fixed can you please re base your PR .

File: fs/fsx.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    The Fsx test is a file system exerciser test
+    '''
+
+    def setUp(self):
+        '''
+        Setup fsx
+        '''
+        smm = SoftwareManager()
+        deps = ['gcc', 'make']
+        for package in deps:
```
Comment (PraveenPenguin) [inline fs/fsx.py]: can this be like for package in ['gcc', 'make'] just want to eliminate extra variable

File: fs/fsx.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            if not smm.check_installed(package) and not smm.install(package):
+                self.error(package + ' is needed for the test to be run')
+
+        fsx = self.fetch_asset(
+            'https://raw.githubusercontent.com/linux-test-project/ltp/'
+            'master/testcases/kernel/fs/fsx-linux/fsx-linux.c')
```
Comment (PraveenPenguin) [inline fs/fsx.py]: it always good to use expire param in fetch_asset module

File: fs/fsx.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        os.chdir(self.srcdir)
+        process.system('gcc -o fsx %s' % fsx, shell=True)
```
Comment (PraveenPenguin) [inline fs/fsx.py]: please use ignore_status=True process.system problem with running command script will error out

**Comment by harish-24** [comment]
@PraveenPenguin Sent a new PR #257 with the changes. Closing this.

**Pull request closed without merging**